### PR TITLE
feat(buttongroup): added playground for buttonGroup

### PR DESCRIPTION
**Pull request by DanSallau**

### CODE REVIEW

File: src/pages/docs/buttongroup.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <Docs
+            mainFile={importString}
+            importedFiles={importedFiles}
+            example={
+                <div className="w-100 d-flex justify-content-center">
+                    {controls?.toolbar ? (
+                        <div className="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
+                            {[...new Array(2)].map((toolBar: undefined, index: number) => (
+                                <div className="m-2" key={`toolbar-${index}`}>
+                                    {renderButton}{" "}
```
Comment (mario-subo) [inline src/pages/docs/buttongroup.tsx:81]: shouldn't this be `renderButton()` ?

Comment (DanSallau) [reply src/pages/docs/buttongroup.tsx:81]: Nope, its a variable.

**Pull request merged**